airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -358,15 +358,14 @@ def execute(self, context) -> Optional[str]:
 
             if len(pod_list.items) == 1:
                 try_numbers_match = self._try_numbers_match(context, pod_list.items[0])
-                final_state, result = self.handle_pod_overlap(
+                final_state, pod_info, result = self.handle_pod_overlap(
```

Comment:
```suggestion
                final_state, _, result = self.handle_pod_overlap(
```
Since we don't use it?

Reply:
It's used on line 368 if the resulting pod isn't successful, no?

Reply:
Oh yeah, missed that!